.env.example

```
@@ -39,3 +39,5 @@ MESSENGER_TRANSPORT_DSN=doctrine://default?auto_setup=0
 ###> symfony/mailer ###
 # MAILER_DSN=null://null
 ###< symfony/mailer ###
+
+INVERTEXTO_API_TOKEN='16517|IkMw30HFuY9kAP08b97w0NJlYtzU4DSa'
```

a varavel ambiente existe mas a secret não pode ficar aqui... PERIGO

```suggestion

INVERTEXTO_API_TOKEN=
```